DROD/FaceWidget.cpp

```diff
@@ -340,9 +340,9 @@ void CFaceWidget::SetReading(const bool bReading)
 }
 
 //****************************************************************************
-void CFaceWidget::SetSleeping()
+void CFaceWidget::SetSleeping(const bool bIsSleeping)
 {
-	facePlayer.bIsSleeping = true;
+	facePlayer.bIsSleeping = bIsSleeping;
 	facePlayer.bIsBlinking = true;
```

Comment:
Is bIsBlinking supposed to be false when bIsSleeping is set to false?

Reply:
Ultimately it does not matter, since it just means that Beethro will have eyes closed for one more FaceWidget frame, but it'll be cleared for future maintainers if it's also set. Updated PR!